packages/react-offix-hooks/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-offix-hooks",
```

Comment:
what will be your approach for react.. helpers or entire offix-react wrapper? name can change?

Reply:
I see this module as being really small and only covering the hooks. I don't think the name will need to change. For now react integrations should/will be delivered by existing libraries out there.